registry-library/library/library.go

```diff
@@ -356,6 +305,274 @@ func PullStackFromRegistry(registry string, stack string, destDir string, option
 	return PullStackByMediaTypesFromRegistry(registry, stack, DevfileAllMediaTypesList, destDir, options)
 }
 
+// DownloadStarterProjectAsDir downloads a specified starter project archive and extracts it to given path
+func DownloadStarterProjectAsDir(path string, registryURL string, stack string, starterProject string, options RegistryOptions) error {
```

Comment:
@kim-tsao Preferably part of this PR, since the link in the template was broken I wasn't sure if this was where the docs were still added. I can add documentation to this file matching the scheme of the others already there as a revision.

Reply:
Also I'll update the broken links for the current PR.

Reply:
@kim-tsao Made changes to https://github.com/devfile/registry-support/blob/main/registry-library/README.md, let me know if these changes look good for the documentation.